tools/notice/build.gradle.kts

```diff
@@ -30,16 +30,20 @@ val noticeDir = project.layout.buildDirectory.dir("notice")
 val includeNoticeLicenseFiles by
   tasks.registering(Sync::class) {
     destinationDir = noticeDir.get().asFile
-    inputs.files(rootProject.layout.files("NOTICE*", "LICENSE*", "version.txt"))
+    inputs.files(rootProject.layout.files("NOTICE", "LICENSE", "version.txt"))
+    inputs.files(
+      rootProject.layout.files(
+        "gradle/built-uber-dists/NOTICE-BINARY-DIST",
+        "gradle/built-uber-dists/LICENSE-BINARY-DIST",
+      )
+    )
     inputs.property("version", project.version)
-    from(rootProject.projectDir) {
-      include("NOTICE-BINARY-DIST", "NOTICE", "LICENSE-BINARY-DIST", "LICENSE")
+    from(rootProject.projectDir.resolve("gradle/built-uber-dists")) {
+      include("NOTICE-BINARY-DIST", "LICENSE-BINARY-DIST")
```

Comment:
`nessie-model-0.102.5-SNAPSHOT.jar` does not have a NOTICE. I think we should put it in all jars, right?

Reply:
I think we have to build with a specific profile. I think we should do that by default.

Reply:
Nope, let's not do unnecessary stuff like that. It's done for releases.

Reply:
It was my mistake with the gradle command. After rebuilding I see LICENSE and NOTICE are included under `META-INF/licenses/org.projectnessie.nessie/nessie-model-0.102.5-SNAPSHOT/`